apps/acnw/content/AboutAmberconNwContent.mdx

```diff
@@ -5,13 +5,12 @@ title:  What is AmberCon NW?
 import { ConfigurationValue, ConventionYear, ConventionsDatesFull, ContactEmail } from 'amber/components'
 import { SlotDescriptionList } from 'amber/components/SlotDescriptionList'
 
-AmberCon Northwest is a fully scheduled role-playing game convention devoted diceless / story-focused / character driven
-RPGs. While a large portion of the games use Erick Wujcik's **Amber Diceless RPG** as their base ruleset, or are set in
-Roger Zelazny's **Amber** universe, it also includes many offerings of other diceless and indie published games, as well
-as a few more traditional games, and even a board game slot.
+AmberCon Northwest is a diceless and indie role-playing convention held annually in Troutdale, Oregon, just east of Portland. 
```

Comment:
I'd like to keep the "a fully scheduled" bit simply because it was added to clearly differentiate ambercon from all of the other types of gaming convention that are still around and cater to pick up games.  Other than that, no objections.

Reply:
Yay!

I chose against "fully scheduled" because imo it is jargon that is only be meaningful to people who already know how the con works, and what it means is explained later on the info page. 

Even so, I don't feel strongly about this, so I'm fine with keeping it in.

Reply:
fair point.  I ended out adding it before I saw your reply, but mostly to trigger a build as the previous one needed to be re-run since there was a bug on the main branch.